projects/angular/src/forms/common/STYLES.label.md

```diff
@@ -0,0 +1,48 @@
+# Label
+
+## Variables
```

Comment:
Change to CSS Custom Properties and document the custom properties (from the `_properties.component-name.scss` instead of the variables.